front/app/components/Unauthorized/index.tsx

```diff
@@ -63,16 +56,8 @@ const Unauthorized = () => {
             {formatMessage(messages.sorryNoAccess)}
           </Text>
           <Box mb="16px">
-            <Button onClick={signUp} text={formatMessage(messages.signUp)} />
+            <Button onClick={signIn} text={formatMessage(messages.signIn)} />
```

Comment:
Fixes 
<img width="1431" alt="DONE - No access signed out" src="https://user-images.githubusercontent.com/16427929/225950948-3ced537a-fe36-4e71-999a-800a1c70eda7.png">

Reply:
Sign in > sign up. And the style of the Log in link was out of line with the rest of our styles.

Reply:
So, now instead of two buttons (Sign up/Sign in) we have only one (Sign in), right? Doesn't the second button have some value? Why are you removing it?

Reply:
Yes, I discussed it with Yaesul. It's uncommon to be asked to sign up instead of log in first. 

Then just log in is fine because there's also a link to sign up in the log in form.

We could add sign up in a link (with correct styles) if people complain, but I don't think it should be on this particular page.